engine/render/Makefile

```diff
@@ -45,9 +56,15 @@ all: $(renderLib) $(vertObjects) $(fragObjects) vulkan-libs
 $(renderLib): $(renderObjects)
 	$(MKDIR) $(call platformpth, $(libDir))
 	$(RM) $(call platformpth, $(renderLib))
-	$(call COPY,$(vendorDir)/glfw/src,$(libDir),libglfw3.a)
-	mv $(call platformpth,$(libDir)/libglfw3.a) $(call platformpth,$(libDir)/librender.a)
-	ar -rcs $(renderLib) $(renderObjects) 
+
+	cd $(libDir) $(THEN) ar -x $(vendorDir)/glfw/src/libglfw3.a
+	cd $(libDir) $(THEN) ar -x $(vendorDir)/zlib/build/lib/libz.a
+	cd $(libDir) $(THEN) ar -x $(libpngBuildDir)
+	cd $(libDir) $(THEN) ar -x $(freetypeLibDir)
+
+	cd $(libDir) $(THEN) ar -qc $(renderLib) $(renderObjects) $(objExtension)
+	cd $(libDir) $(THEN) rm $(objExtension)
+	$(symdefDelete)
```

Comment:
Can you explain the changes to the archive commands here?

Reply:
Windows doesn't allow us to use the previous command structure - its commands are just not able to take all the directories and merge them into a single .a file. As a result we need to manually unpack each one and re-pack them. I've tried multiple ways to get Windows building with this but this is the only method that seems to work. If you'd like to give it a shot then you're most welcome.